animated-scroll-to.js

```diff
@@ -6,6 +6,11 @@
   function __ANIMATE_SCROLL_TO(desiredOffset) {
     var userOptions = arguments.length <= 1 || arguments[1] === undefined ? {} : arguments[1];
 
+    if(desiredOffset instanceof HTMLElement){
+      var scrollTop = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop;
```

Comment:
Please add spaces before `(` and after `)`

It is enough to use `window.scrollY || document.documentElement.scrollTop`, `pageYOffset` is just an alias for `scrollY`.